apps/forms/form-dialog-alert/project.json

```diff
@@ -20,7 +20,10 @@
           "apps/forms/form-dialog-alert/src/favicon.ico",
           "apps/forms/form-dialog-alert/src/assets"
         ],
-        "styles": ["apps/forms/form-dialog-alert/src/styles.scss"],
+        "styles": [
+          "apps/forms/form-dialog-alert/src/styles.scss",
+          "./node_modules/@angular/material/prebuilt-themes/indigo-pink.css"
```

Comment:
When I served untouched challenge, I didn't have an error in browser console without the theme installed.  Could be necessary to add challenge anyway?

Reply:
Yes the error (warn to be precise) only occurs if you import `MatDialogModule` for example. I looked at the challange again, in the tips section there is a hint with a nicer solution, so i'll refactor this.

> Use the Material CDK Dialog or Overlay - https://material.angular.io/cdk/ (dont forget to add @import '@angular/cdk/overlay-prebuilt.css' in style.sccss)

Reply:
And what do you mean with "Could be necessary to add challenge anyway?"

Reply:
yes we can add it to the starter file. That's not part of the challenge to find out how to import the theme